scenarios/basics/location/SPECIFICATION.md

```diff
@@ -0,0 +1,450 @@
+Distance: 3 meters
+-------------------------
+Place Name: POINT_OF_INTEREST
+Address: Timberwood Tree Service, Market St, San Francisco, CA 94103, United States
+Distance: 3 meters
+-------------------------
+Place Name: POINT_OF_INTEREST
+Address: Wonderbread5, Market St, San Francisco, CA 94103, United States
+Distance: 3 meters
+-------------------------
+Place Name: POINT_OF_INTEREST
+Address: Weaving Stories Therapy, Market St, San Francisco, CA 94103, United States
+Distance: 3 meters
+-------------------------
+Place Name: POINT_OF_INTEREST
+Address: Chef Greg / Personal Chef, Market St, San Francisco, CA 94103, United States
+Distance: 3 meters
+-------------------------
+Place Name: POINT_OF_INTEREST
+Address: Illuminate Bodywork, Market St, San Francisco, CA 94103, United States
+Distance: 3 meters
+-------------------------
+Place Name: POINT_OF_INTEREST
+Address: Book My Limo Trip, Market St, San Francisco, CA 94103, United States
+Distance: 3 meters
+-------------------------
+Place Name: POINT_OF_INTEREST
+Address: Cardona Landscaping, Market St, San Francisco, CA 94103, United States
+Distance: 3 meters
+-------------------------
+Place Name: POINT_OF_INTEREST
+Address: C.H. Burnham Construction, Market St, San Francisco, CA 94103, United States
+Distance: 3 meters
+-------------------------
+Place Name: POINT_OF_INTEREST
+Address: Edward Dougherty, Market St, San Francisco, CA 94103, United States
+Distance: 3 meters
+-------------------------
+Place Name: POINT_OF_INTEREST
+Address: Ripertonwest Consulting, Market St, San Francisco, CA 94103, United States
+Distance: 3 meters
+-------------------------
+Place Name: POINT_OF_INTEREST
+Address: FedEx Office, 1967 Market St, San Francisco, CA 94103, United States
+Distance: 4 meters
+-------------------------
+--------------------------------------------------------------------------------
+12. Delete the AWS Location Services resources.
+Would you like to delete the AWS Location Services resources? (y/n)
+y
+The map MyMap40 was deleted.
+The key MyLocationKey40 was deleted.
+The geofence collection AWSLocationCollection40 was deleted.
+The tracker geoTracker40 was deleted.
+The route calculator AWSRouteCalc40 was deleted.
+
+Enter 'c' followed by <ENTER> to continue:
+c
+Continuing with the program...
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+ This concludes the AWS Location Service scenario.
+--------------------------------------------------------------------------------
+
+```
+
+## SOS Tags
+
+The following table describes the metadata used in this Basics Scenario.
+
+| action                 | metadata file             |     metadata key                    |
+|------------------------|---------------------------|-------------------------------------|
+| `createKey`            | location_metadata.yaml    |location_CreateKey                   |
```

Comment:
Is the hello example not listed?

Reply:
It is listed:

| `hello`                | location_metadata.yaml    |location_Hello